app/frontend/i18n/i18n.ts

```diff
@@ -137,6 +137,205 @@ export const resources = {
+        landing: {
+          title: "Building Permit Hub",
+          intro:
+            "Co-created with a variety of pilot local jurisdictions for the people of B.C. to help create more homes faster.",
+          easilyUpload: "Easily upload all your required building permitting information such as pdf files",
+          bestPractices: "Get best-practices from  provincial and local jurisdictions",
+          easyToFollow: "Easy to follow instructions of what is required for your building permit application",
+          accessMyPermits: "Access My Housing Building Permits",
+          accessExplanation:
+            "Digital Building Permit Account uses the same or different login as BCeID. Need to explain this to users clearly what they’re logging in with.",
+          whoForTitle: "Who is this for?",
+          whoFor: [
+            "I want to build a houseplex",
+            "I want to build a small building on my property",
+            "Industry professionals",
+            "Building Permits in BC for Housing",
+          ],
+          iNeed: "What do I need?",
+          whyUseTitle: "Why use this tool?",
+          whyUse:
+            "This is a housing building permitting tool pilot to help all communities in BC receive and process building permit applications faster and more efficiently.  This tool links into the single application portal for Provincial natural resource permits that may also be required for some housing building permit applications.",
+          iNeedLong: "What do I need for a housing building permit?",
+          reqsVary:
+            "Permit requirements vary by local jurisdiction and depend on the geography of the surrounding location.",
+          whereTitle: "Where",
+          findAuthority: "Find your local building permitting authority.",
+          locationOr: "Location or Civic Address",
+          withinXRiver: "Within x km of a river",
+          withinXForest: "Within x km of a forest",
+          withinXProtected: "Within x km of a protected land",
+          whatType: "What type of housing are you building?",
+          dontSee: "Don't see the type that you're looking for?",
+          whenNotNecessaryQ: "When is a permit not necessary?",
+          whenNotNecessaryA:
+            "Projects that are for the interior of your home, minor repairs. Things like fence, sheds may depend on local jurisdiction and geography.",
+          expectQ: "What can I expect?",
+          expectA: "After submitting your permit application through this tool, lorem ipsum dolor sit amet.",
+          createdQ: "Why was this tool created?",
+          createdA:
+            "Becoming a North American leader of digital permitting and construction by digitally integrating permit systems and tools across the housing development sector across B.C. is a commitment of the 2023 Ministry of Housing Homes for People Plan.",
+        },
+        ui: {
+          back: "Back",
+          show: "Show",
+          hide: "Hide",
```

Comment:
This looks like a rebase/merge err
![Screenshot 2024-01-17 at 12 32 08 PM](https://github.com/bcgov/HOUS-permit-portal/assets/32022335/dcaa651e-146c-4fb0-a711-006e286b2293)
or. All these keys are under requirementsLibrary when it shouldn't be